### ISSUE TITLE
Bug? 

### ISSUE THREAD

**Issue by number12** [Samgisaninja/SuccessionRestore#21]
ios 12.1.2  iphone 8 plus. Been trying all the successions and the latest deb of 1.1.2 acts like it is working after rebooting and trying over after 1-3 times. Says it "successfully restored" or something along that lines and shows the apple logo and a progress bar, but after a couple seconds it resprings and then I am brought back to my homescreen unjailbroken. I reboot and gotta reinstall everything even though trying to do a full restore. Also sometimes it gets the mounting bug

**Comment by Samgisaninja** [comment]
1. Duplicate of #19, also I believe #19 may be fixed, just waiting for confirmation
2. Make sure “use test mode” and “only restore system data” are OFF

**Comment by number12** [comment]
> 1. Duplicate of #19, also I believe #19 may be fixed, just waiting for confirmation
> 2. Make sure “use test mode” and “only restore system data” are OFF

thanks for the reply and hard work. Happy to know im not only one seeing this effect. and yes they are both off. Anyway I can send you some type of log or something if there is one?

**Comment by yp261** [comment]
Yes, if logging is enabled you can head to /var/mobile/ and find a log there with the name of the tweak. which is Succession

to enable logging, just go to settings and check it. It should be enabled from the start so go and take a look if there is something there.

**Comment by number12** [comment]
alright cool here is my log hopefully it can help. I just tried with the latest 1.1.4 and still does the apple logo and progress bar, resprings to homescreen passcode
[succession.log](https://github.com/Samgisaninja/SuccessionRestore/files/3115421/succession.log)

**Comment by Samgisaninja** [comment]
Hmm, this is really strange, Succession is actually working perfectly, but the last thing Succession does is tell iOS to run “Erase all content and settings”, and that’s failing. 

I don’t know if there’s anything I can do to fix that, if anyone else gets this issue, please comment, but for now, once the restore is complete, just run erase all content and settings manually.

**Comment by yp261** [comment]
> Hmm, this is really strange, Succession is actually working perfectly, but the last thing Succession does is tell iOS to run “Erase all content and settings”, and that’s failing.
> 
> I don’t know if there’s anything I can do to fix that, if anyone else gets this issue, please comment, but for now, once the restore is complete, just run erase all content and settings manually.

remember **not to jailbreak** before running "erase all content and settings" because you will bootloop yourself. :)

**Comment by Samgisaninja** [comment]
This issue is (finally) fixed in 1.4.2, sorry that took so long
